plugins/ruby-foreman-ansible/debian/rules

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/make -f
+# -*- makefile -*-
+# Sample debian/rules that uses debhelper.
+# This file was originally written by Joey Hess and Craig Small.
+# As a special exception, when this file is copied by dh-make into a
+# dh-make output file, you may use that output file without restriction.
+# This special exception was added by Craig Small in version 0.37 of dh-make.
+
+# Uncomment this to turn on verbose mode.
+#export DH_VERBOSE=1
+
+PLUGIN = foreman_ansible
+
+build:
```

Comment:
This whole section (and PLUGIN) can be removed if there's no precompilation happening, see ruby-foreman-templates.